content/05.treat.md

```diff
@@ -61,7 +61,7 @@ studies have attempted to create more interpretable deep models, either
 specifically for healthcare or as a general procedure for deep learning (see
 Discussion).
 
-#### Predicting patient trajectories
+#### Predicting patient trajectories from treatments
```

Comment:
I again prefer the existing sub-heading.  I don't know these papers well, but I suspect some of the are making predictions from general clinical data, not just treatments.

Reply:
Sure.